applications/launchpad/gui-react/src/components/TitleBar/index.tsx

```diff
@@ -26,10 +29,14 @@ import {
 } from './styles'
 import { TitleBarProps } from './types'
 
-const TitleBar = ({ drawerViewWidth = '50%' }: TitleBarProps) => {
+const TitleBar = ({
+  drawerViewWidth = '50%',
+  mode = 'help',
```

Comment:
NIT: Because `mode` controls only one thing (show/hide the Settings button), maybe it would be better to change `mode = 'help'` into `hideSettingsBtn = false`. Then, it will be easier figure out how to configure props of TitleBar if we would like to use it somewhere else.

Reply:
Will get this in the next PR I'm working on just now 👍